src/ucp/rndv/rndv_rkey_ptr.c

```diff
@@ -100,8 +100,13 @@ ucp_rndv_rkey_ptr_query_common(const ucp_proto_query_params_t *params,
     const ucp_proto_rndv_rkey_ptr_priv_t *rpriv = params->priv;
 
     ucp_proto_default_query(params, attr);
-    attr->lane_map = UCS_BIT(rpriv->spriv.super.lane) |
-                     UCS_BIT(rpriv->ack.lane);
+
+    ucs_assertv(rpriv->spriv.super.lane != UCP_NULL_LANE, "lane=%d",
+                rpriv->spriv.super.lane);
+    attr->lane_map = UCS_BIT(rpriv->spriv.super.lane);
+    if (rpriv->ack.lane != UCP_NULL_LANE) {
+       attr->lane_map |= UCS_BIT(rpriv->ack.lane);
+    }
```

Comment:
This change is required since `ack.lane` can be equal to  255 (`UCP_NULL_LANE`) so after increasing `lane_map` size to 64 it started to set 63rd bit in lane_map.

Reply:
i think `(1ul << 256)` is anyway undefined behavior as 256 >= 64.

Reply:
i was experimenting on similar case and got unpredictable result, i guess this should be ok